packages/console/src/components/Table/index.tsx

```diff
@@ -80,21 +97,31 @@ const Table = <
                     </td>
                   </tr>
                 )}
-                {data?.map((row) => (
-                  <tr
-                    key={row[rowIndexKey]}
-                    className={classNames(onClickRow && styles.clickable)}
-                    onClick={() => {
-                      onClickRow?.(row);
-                    }}
-                  >
-                    {columns.map(({ dataIndex, colSpan, className, render }) => (
-                      <td key={dataIndex} colSpan={colSpan} className={className}>
-                        {render(row)}
-                      </td>
-                    ))}
-                  </tr>
-                ))}
+                {data?.map((row) => {
+                  const rowClickable = isRowClickable(row);
+
+                  const onClick = conditional(
+                    rowClickable &&
+                      rowClickHandler &&
+                      (() => {
+                        rowClickHandler(row);
+                      })
+                  );
+
+                  return (
+                    <tr
+                      key={row[rowIndexKey]}
+                      className={classNames(rowClickable && styles.clickable)}
+                      onClick={onClick}
```

Comment:
```
onClick = {conditional(isRowClickable(row) && rowClickHandler && () =>{ rowClickHandler(row) })}
```

Reply:
Prefer to keep the temporary variable to keep the HTML code tidy.